tests/reset_workflow.go

```diff
@@ -651,6 +652,113 @@ func (s *FunctionalSuite) testResetWorkflowReapplyBuffer(
 
 }
 
+func (s *FunctionalSuite) TestUpdateMessageInLastWFT() {
+	s.dynamicConfigOverrides = map[dynamicconfig.Key]interface{}{
+		dynamicconfig.FrontendEnableUpdateWorkflowExecutionAsyncAccepted: true,
+	}
+	tv := testvars.New(s.T().Name())
+	workflowId := "wfid"
+	workflowTypeName := "wftype"
+	taskQueueName := "tq"
+	identity := "worker1"
+	workflowType := &commonpb.WorkflowType{Name: workflowTypeName}
+	taskQueue := &taskqueuepb.TaskQueue{Name: taskQueueName, Kind: enumspb.TASK_QUEUE_KIND_NORMAL}
```

Comment:
I see you are using `tv` in this test. You don't need this vars, you can get values from `tv`.